test/lib/move.spec.ts

```diff
@@ -27,11 +28,8 @@ describe('lib/move.ts', () => {
     const contactType = Config.getContactType('c_community_health_unit');
     const sessionCache = new SessionCache();
     
-    const actual = await MoveLib.move(formData, contactType, sessionCache, chtApi());
-    expect(actual.command).to.include('--contacts=chu-id');
-    expect(actual.command).to.include('--parent=to-sub');
-    expect(actual.command).to.include('--url=http://username:password@domain.com', actual.command);
-
+    const actual = await MoveLib.move(formData, contactType, sessionCache, chtApi(), mockQueueManager());
+    expect(actual.jobsBoardUrl).to.exist;
```

Comment:
can you add back these assertions? I think these are good tests

Reply:
the  `command` is no more available here, since we don't need anymore?